logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/HttpSpringLogger.java

```diff
@@ -53,6 +54,20 @@ public static void start(final Logger logger, final Logger auditor, final HttpSe
         start(logger, auditor, request, null);
     }
 
+    public static HttpSpringLogger startAsync(final Logger logger, final Logger auditor, final HttpServletRequest request) {
+        return startAsync(logger, auditor, request, null);
+    }
+
+    public static HttpSpringLogger startAsync(final Logger logger, final Logger auditor, final HttpServletRequest request, final String requestBody) {
```

Comment:
can you share example on usage? how do you overcome if the flow includes multiple threads for the same request?